src/java.desktop/windows/native/libawt/java2d/d3d/D3DBadHardware.h

```diff
@@ -54,6 +54,9 @@ static const ADAPTER_INFO badHardware[] = {
     // All Intel Chips.
     { 0x8086, ALL_DEVICEIDS, NO_VERSION, OS_ALL },
 
+    // Microsoft Basic Render Driver (as maybe used in VMs such as VirtualBox)
+    { 0x1414, 0x008c, NO_VERSION, OS_ALL },
```

Comment:
Any particular reason for not putting ALL_DEVICEIDS?

Reply:
er, yeah, because it would be wrong. If Microsoft had a graphics card of their own, say in a Surface Pro,
all device ids would disable that too .. and I have no reason to do that without evidence that all MS drivers
for all theoretical MS hardware will have problems